evcxr/src/command_context.rs

```diff
@@ -586,6 +600,29 @@ Panic detected. Here's some useful information if you're filing a bug report.
         out.push_str("</table>");
         out
     }
+
+    fn var_type(&self, args: &Option<String>) -> Result<EvalOutputs, Error> {
+        let args = if let Some(x) = args {
+            x.trim()
+        } else {
+            bail!("Variable name required")
+        };
+
+        let mut out = None;
+        for (var, ty) in self.eval_context.variables_and_types() {
+            if var == args {
+                out = Some(ty.to_owned());
+                break;
+            }
+        }
```

Comment:
Alternatively one could add a public function to eval_context to lookup a single variable. Not sure what is better, so I went with less intrusive changes and did like in vars_as_text.